include/ck_tile/ops/gemm/kernel/gemm_kernel.hpp

```diff
@@ -56,6 +56,20 @@ struct GemmHostArgs : public GemmProblem
     index_t k_batch;
 };
 
+struct GemmKernelArgs
```

Comment:
I have a mixed feelings about this change, since this exposes internal Kernel arguments. I think I'd prefer to keep it in-class.

Reply:
I have the same opinion, but we need to know the size of `GemmKernelArgs` struct, because we are allocating workspace memory before creating `Kernel` class, so that's why I have exposed this class.